stable/redis-ha/templates/redis-ha-configmap.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-configmap
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+data:
+  redis.conf: |
+    dir "/data"
+    {{- range $key, $value := .Values.redis.config }}
+    {{ $key }} {{ $value }}
+    {{- end }}
+
+  sentinel.conf: |
+    dir "/data"
+    {{- $root := . -}}
```

Comment:
could we have a space here to configure other sentinel flags? like 'protected-mode no' sort of things I mean

Reply:
This is something I was struggling with as well. What I ended up doing was allowing anything under then `.Values.sentinel.config` to be passed to the sentinel.conf. So what you're asking is already possible and simply needs to be added to the values.yaml prior to install.
https://github.com/helm/charts/pull/7323/files#diff-475a26a8007c70fc590900868d83e50eR36

However the problem currently is (and any solutions/input is welcome) is that this will only work if you set all your values prior to install and currently doesn't add values on upgrade. The init script I wrote will only copy over the sentinel.conf if there isn't a pre-existing one and only update config options that were there during initial install or manually added. This was done because Sentinel is continually updating the sentinel.conf live as slaves go up/down, masters get promoted, new settings get manually set, etc.

Reply:
great, thanks for the explanation!